dotnet/src/SemanticKernel/Configuration/KernelConfigExtensions.cs

```diff
@@ -0,0 +1,199 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using Microsoft.SemanticKernel.AI.ChatMessageGeneration;
+using Microsoft.SemanticKernel.AI.Embeddings;
+using Microsoft.SemanticKernel.AI.ImageGeneration;
+using Microsoft.SemanticKernel.AI.OpenAI.Services;
+using Microsoft.SemanticKernel.AI.TextCompletion;
+using Microsoft.SemanticKernel.Diagnostics;
+
+namespace Microsoft.SemanticKernel.Configuration;
```

Comment:
These extensions are all about OpenAI... can we isolate this to the /OpenAI/ folder and namespace?
Better separation of interests, allows us to more easily move the whole OpenAI service library to its own project

Reply:
my thought was that this would keep growing to include stable diffusion and others. In that case would you prefer having multiple extension classes, under each brand/provider?